src/nodes/Table/TableCell.js

```diff
@@ -30,4 +32,32 @@ export default TableCell.extend({
 			},
 		}
 	},
+
+	addProseMirrorPlugins() {
+		return [
+			new Plugin({
+				props: {
+					handlePaste: (view, event, slice) => {
+						if (slice.content.childCount > 1) {
+							const state = view.state
+							const childNodes = []
+							for (let i = 0; i < slice.content.childCount; i++) {
+								if (i) {
```

Comment:
```suggestion
								if (i === 0) {
```

This would make the check more obvious in my opinion.